src/core/Banner/index.tsx

```diff
@@ -48,6 +49,7 @@ const Banner = forwardRef<HTMLDivElement, BannerProps>(function Banner(
           <Icon sdsIcon="infoCircle" sdsSize="l" sdsType="static" />
         </IconWrapper>
         <Text>{text}</Text>
+        {props.children}
```

Comment:
This leaves some confusion around what will get rendered when text and children are passed in. It would be more clear to throw an error if the user passes both `text` and `children` props.